R/derive_var_anrind.R

```diff
@@ -1,9 +1,11 @@
 #' Derive Reference Range Indicator
 #'
 #' @param dataset The input dataset
+#' @param use_a1hia1lo Boolean value indicating whether to use `A1H1` and `A1LO` in
```

Comment:
```suggestion
#' @param use_a1hia1lo A logical value indicating whether to use `A1H1` and `A1LO` in
```

Think everywhere else in our docs we use the term logical in favor of boolean